poll/poll.py

```diff
@@ -811,31 +826,31 @@ def vote(self, data, suffix=''):
         choices = self.get_choices()
         if choices and not self.private_results:
             result['success'] = False
-            result['errors'].append("You have already voted in this poll.")
+            result['errors'].append(self.ugettext("You have already voted in this poll."))
 
         if not choices:
             # Reset submissions count if choices are bogus.
             self.submissions_count = 0
 
         if not self.can_vote():
             result['success'] = False
-            result['errors'].append('You have already voted as many times as you are allowed.')
+            result['errors'].append(self.ugettext('You have already voted as many times as you are allowed.'))
 
         # Make sure the user has included all questions, and hasn't included
         # anything extra, which might indicate the questions have changed.
         if not sorted(data.keys()) == sorted(questions.keys()):
             result['success'] = False
             result['errors'].append(
-                "Not all questions were included, or unknown questions were "
-                "included. Try refreshing and trying again."
+                self.ugettext("Not all questions were included, or unknown questions were "
+                              "included. Try refreshing and trying again.")
```

Comment:
@asadiqbal08 Please adjust formatting as described in ["Breaking Long Lines"](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#breaking-long-lines).